Guidelines/Guidelines.md

```diff
@@ -5,3 +5,54 @@
     with no Invariant Sections, no Front-Cover Texts, and no Back-Cover Texts.
     A copy of the license is included in the section entitled "GNU
     Free Documentation License".
+
+# PSC Event Guidelines
```

Comment:
I think, it will be better to write "PLLUG Summer Camp" instead of "PSC" in headline.